certified-connectors/Autenti E-Signature Standard Workflow Connector/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1083 @@
+{
```

Comment:
Kindly fix below swagger validation errors. 


##[error]The 'description' property is required. ValuePath: /paths/~1document-processes~1{documentProcessId}~1files/post/parameters/0
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1document-processes~1{documentProcessId}~1files/post/parameters/1
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1document-processes~1{documentProcessId}~1files/post/parameters/2
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1document-processes~1{documentProcessId}~1files/post/parameters/3
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1document-processes~1{documentProcessId}~1actions/post/parameters/0
##[error]The 'description' property is required. ValuePath: /paths/~1document-processes~1{documentProcessId}~1actions/post/parameters/0
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1document-processes~1{documentProcessId}~1files~1{fileId}~1content/get/parameters/0
##[error]The 'description' property is required. ValuePath: /paths/~1document-processes~1{documentProcessId}~1files~1{fileId}~1content/get/parameters/0
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1document-processes~1{documentProcessId}~1files~1{fileId}~1content/get/parameters/1
##[error]The 'description' property is required. ValuePath: /paths/~1document-processes~1{documentProcessId}~1files~1{fileId}~1content/get/parameters/1
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1applications~1callbacks/post/parameters/0
##[error]The 'description' property is required. ValuePath: /paths/~1applications~1callbacks/post/parameters/0
##[error]The 'default' response should not have schema definition. Schemas should be defined on expected responses only. ValuePath: /paths/~1document-processes/get/responses/default
##[error]The 'default' response should not have schema definition. Schemas should be defined on expected responses only. ValuePath: /paths/~1document-processes/post/responses/default
##[error]The 'default' response should not have schema definition. Schemas should be defined on expected responses only. ValuePath: /paths/~1document-processes~1{documentProcessId}~1parties/post/responses/default
##[error]The 'default' response should not have schema definition. Schemas should be defined on expected responses only. ValuePath: /paths/~1document-processes~1{documentProcessId}~1actions/post/responses/default
##[error]The connector category is invalid. Available categories are: [AI, Business Management, Business Intelligence, Collaboration, Commerce, Communication, Content and Files, Finance, Data, Human Resources, Internet of Things, IT Operations, Lifestyle and Entertainment, Marketing, Productivity, Sales and CRM, Security, Social Media, Website]. ValuePath: /x-ms-connector-metadata/2/propertyValue
##[error]The maximum allowed number of categories for the connector is '2'. ValuePath: /x-ms-connector-metadata/2/propertyValue

Reply:
Done at commit 897512aad8dcc50e54f3e6db61b12e94272fa1ee